tools/bin/commands/build

```diff
@@ -269,6 +269,7 @@ MODULES=(
   "server:common extension integration connector"
   "s2i:common extension integration connector server"
   "test:common extension integration connector server s2i"
+  "teiid-syndesis"
```

Comment:
I would just call these "dv"

Reply:
So your going to rename teiid-syndesis to dv when u port it into syndesis?

Reply:
It is just repo name we do not need to port anything. if you call as "dv" it will match code references during the build to the runtime names, so that it will be consistent across. IDK, but you could simply called it "dv" sub-module too, but reference `teiid-syndesis` repo.

Reply:
the goal, i thought, was to move teiid-syndesis into syndesis.  After talking with zoran, he indicated that it should land under the app directory.    Unless you plan on changing teiid-syndesis to something else, the scripts will specifically reference this folder name.   I was staying with teiid-syndesis so that a different name doesn't have to remembered or changed later.

Reply:
i'll rename to dv.